src/Persistence/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Domain.Models;
+using Persistence.Contexts;
+using Services.Interfaces.Repositories;
+
+namespace Persistence.Repositories
+{
+    public class UserRepository : Repository<User>, IUserRepository
+    {
+        private readonly ApplicationDbContext _db;
+
+        public UserRepository(ApplicationDbContext db) : base(db)
+        {
+            _db = db;
+        }
+    }
+}
```

Comment:
If you need to edit something and use `DbContext` directly, please make it `protected` in `Repository<T>`. It would make it easier.

Reply:
Done